src/com/company/Main.java

```diff
@@ -17,109 +15,105 @@
-        pages.put(PageName.HOME,new HomePage(userService,productService));
-        pages.put(PageName.PRODUCT_SALE,new ProductSalePage(userService,productService));
-        pages.put(PageName.USER_CREATE,new UserCreatePage(userService,productService));
-        pages.put(PageName.PRODUCT_CREATE,new ProductCreatePage(userService,productService));
-        pages.put(PageName.USER_LIST,new UserListPage(userService,productService));
-        pages.put(PageName.PRODUCT_LIST,new ProductListPage(userService,productService));
-        
-        TestPage testPage=new TestPage(userService,productService);
+        pages.put(PageName.LOGIN, new LoginPage(userService, productService));
+        pages.put(PageName.HOME, new HomePage(userService, productService));
+        pages.put(PageName.PRODUCT_SALE, new ProductSalePage(userService, productService));
+        pages.put(PageName.USER_CREATE, new UserCreatePage(userService, productService));
+        pages.put(PageName.PRODUCT_CREATE, new ProductCreatePage(userService, productService));
+        pages.put(PageName.USER_LIST, new UserListPage(userService, productService));
+        pages.put(PageName.PRODUCT_LIST, new ProductListPage(userService, productService));
+
+        TestPage testPage = new TestPage(userService, productService);
         testPage.setOrderService(orderService);
-        pages.put(PageName.TEST,testPage);
+        pages.put(PageName.TEST, testPage);
 
-        OrderPage orderPage= new OrderPage(userService,productService);
+        OrderPage orderPage = new OrderPage(userService, productService);
         orderPage.setOrderService(orderService);
-        pages.put(PageName.ORDER,orderPage);
-        OrderViewPage orderViewPage = new OrderViewPage(userService,productService);
+        pages.put(PageName.ORDER, orderPage);
+        OrderViewPage orderViewPage = new OrderViewPage(userService, productService);
         orderViewPage.setOrderService(orderService);
-        pages.put(PageName.ORDER_VIEW,orderViewPage);
+        pages.put(PageName.ORDER_VIEW, orderViewPage);
         /**
          * Varsayılan kullanıcıların bilgilerinin oluşturulması
          */
         //User type 0 =Admin yetkisi 1 =kasiyer
-        User userAdmin= new User(1,"admin admin","admin","adminadmin",0);
-        User user= new User(2,"user user","useruser","useruser",1);
-        User userCustomer= new User(3,"customer customer","custom","custom",2);
+        User userAdmin = new User("admin admin", "admin", "adminadmin", UserType.ADMIN);
+        User user = new User("user user", "useruser", "useruser", UserType.EMPLOYEE);
+        User userCustomer = new User("customer customer", "custom", "custom", UserType.CUSTOMER);
 
         userService.createUser(userAdmin);
         userService.createUser(user);
         userService.createUser(userCustomer);
-        DB.currentLoginedUser=userCustomer;
+        DB.currentLoginedUser = userCustomer;
```

Comment:
static olan DB listesine currentLoginedUser olarak userCustomer seçili olarak geliyor. Main içerisinde page render edilirken yapılan null kontrolüne girmediği için Test sayfasında loop'da kalınıyor.

Reply:
Unutmuşum ben onu değiştirmeyi hocam sağolasın teşekkür ederim